apps/perf-test/perfLearnings.txt

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
Maybe you can put this and the TODOs from this PR to a readme instead of a txt?

Reply:
Will remove.